cmd/kops/integration_test.go

```diff
@@ -228,6 +228,7 @@ const (
 	canalAddon   = "networking.projectcalico.org.canal-k8s-1.25"
 	ciliumAddon  = "networking.cilium.io-k8s-1.16"
 	flannelAddon = "networking.flannel-k8s-1.25"
+	kindnetAddon = "networking.kindnet-k8s-1.12"
```

Comment:
This should indicate the kops/k8s version that is introduced the change. Would you mind changing to 1.32?
```suggestion
	kindnetAddon = "networking.kindnet-k8s-1.32"
```

Reply:
sure